projects/core/src/lib/users/user.service.ts

```diff
@@ -120,6 +130,28 @@ export class UserService {
         return this.getSessionTokenStream().pipe(first());
     }
 
+    getSessionQuota(): Observable<QuotaDict> {
+        return this.getSessionTokenStream().pipe(mergeMap((sessionToken) => this.backend.getQuota(sessionToken)));
+    }
+
+    getBackendInfoStream(): Observable<BackendInfoDict | undefined> {
+        return this.backendInfo$;
+    }
+
+    isBackendFeatureEnabled(feature: string): Observable<boolean> {
+        return this.getBackendInfoStream().pipe(
+            map((backendInfo) => {
+                if (!backendInfo) {
+                    return false;
+                }
+                if (!backendInfo.features) {
+                    return false;
+                }
+                return backendInfo.features.includes(feature);
+            }),
+        );
+    }
```

Comment:
I'm not sure if we should rely on that.
If we do, we should not make it optional to output this.
But it was intended just for debugging purposes.

Reply:
we should somehow expose backend capabilities

Reply:
i removed it [e46564b](https://github.com/geo-engine/geoengine-ui/pull/586/commits/e46564b203836f6ce2baf1a8d4eec605fc520333)